### PR TITLE
Create home screen

### PR DESCRIPTION
**Pull request by nguyenvanhieu97**
https://edu-redmine.framgia.vn/issues/5085

### CODE REVIEW

**Comment by nguyenvanhieu97** [comment]
ready

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,13 +6,23 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <TextView
-        android:layout_width="wrap_content"
+    <android.support.v4.view.ViewPager
+        android:id="@+id/viewpager"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main.xml]:
```suggestion
        android:id="@+id/view_pager"
```

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                </android.support.v7.widget.Toolbar>
+
+            </android.support.design.widget.CollapsingToolbarLayout>
+
+        </android.support.design.widget.AppBarLayout>
+
+        <android.support.design.widget.FloatingActionButton
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="-18dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_home.xml]: hardcode

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">MovieDB</string>
+    <string name="favorite">Favorite</string>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/values/strings.xml]: 3 -7 các file trong strings.xml theo convention 3.2.2:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md

**Comment by nguyenvanhieu97** [comment]
ready

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">MovieDB</string>
+    <string name="title_favorite">Favorite</string>
+    <string name="title_setting">Setting</string>
+    <string name="title_home">Home</string>
+    <string name="genres">Genres</string>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="title_genres">Genres</string>
    hoăc:
    <string name="text_genres">Genres</string>
```

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,3 +1,8 @@
...
+    <string name="view_all">View all</string>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/values/strings.xml]: tương tự

**Pull request merged**